acceptancetests/assess_model_migration_versions.py

```diff
@@ -67,7 +67,7 @@ def assess_model_migration_versions(stable_bsm, devel_bsm, args):
                 'version-migration',
                 resource_contents)
             migration_target_client = migrate_model_to_controller(
-                test_stable_model, devel_client)
+                test_stable_model, stable_client, devel_client)
```

Comment:
Ah, this is because it's using the wrong client?

Reply:
`migrate_model_to_controller` signature is `(model, src_client, dst_client)` (no defaults!). And both this and the invocation below it used 2 args so I changed the args based on the comment above it.